hub/core/chunk_engine/read.py

```diff
@@ -4,12 +4,21 @@
 
 from .chunker import join_chunks
 
+from hub import constants
 from hub.util.keys import get_meta_key, get_index_map_key
 
 from hub.core.typing import StorageProvider
 from typing import Callable, List, Union
 
 
+def read_tensor_meta(key: str, storage: StorageProvider):
```

Comment:
how should we handle the naming convention of tensor here? i like calling the under the hood stuff tensors also, but there's a collision for `write_array`/`read_array` where they are literally passing arrays. or do you not see this as a problem?

Reply:
Let's stick to `array` for `np.ndarray` and `tensor` for `hub.Tensor`. I think we should be fine with what we have now, but this feels like the natural way to proceed.

Reply:
cool